test/utility/misc.c

```diff
@@ -601,6 +603,34 @@ testutil_time_us(WT_SESSION *session)
     return ((uint64_t)ts.tv_sec * WT_MILLION + (uint64_t)ts.tv_nsec / WT_THOUSAND);
 }
 
```

Comment:
BTW, workgen also has a pareto function (`pareto_calculation`) that should probably be changed to use this.